mobile/android/android-application-manager.ts

```diff
@@ -102,4 +114,8 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
 
 		return applicationViews;
 	}
+
+	public getFullyQualifiedActivityRegex(): RegExp {
+		return /([A-Za-z]{1}[A-Za-z\d_]*\.)*[A-Za-z][A-Za-z\d_]*\/([a-z][a-z_0-9]*\.)*[A-Z_$]($[A-Z_$]|[$_\w_])*/;
```

Comment:
I didn't want to split the regex because of the escaping, but if it's done like so in other places, I can also do it here.